org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -156,6 +149,28 @@ public void start(BundleContext bundleContext) throws Exception {
 		configureProxy();
 	}
 
+	private void setPreferenceNodeId() {
+		Bundle bundle = Platform.getBundle(JDT_UI_PLUGIN);
+		if (bundle != null && bundle.getState() != Bundle.ACTIVE) {
```

Comment:
Please add a comment how this is a hack to ensure unit tests work in Eclipse and CLI builds on Mac